R/globals.R

```diff
@@ -0,0 +1 @@
+utils::globalVariables(c("."))
```

Comment:
What does this do btw?

Reply:
Variables such as those used in the various tidyverse functions that use lazy evaluation end up being flagged as global variables in the CI. Not sure why this wasn't flagged before, but a few new warnings and errors appeared the last time I pushed. Possibly R CMD check has been updated?